samples/user-interface/appwidgets/src/main/java/com/example/platform/ui/appwidgets/glance/weather/WeatherGlanceWidgetReceiver.kt

```diff
@@ -30,24 +34,11 @@ import androidx.glance.appwidget.GlanceAppWidgetReceiver
 class WeatherGlanceWidgetReceiver : GlanceAppWidgetReceiver() {
     override val glanceAppWidget = WeatherGlanceWidget()
 
-    /**
-     * Called when the first instance of the widget is placed. Since all instances share the same
-     * state, we don't need to enqueue a new one for subsequent instances.
-     *
-     * Note: if you would need to load different data for each instance you could enqueue onUpdate
-     * method instead. It's safe to call multiple times because of the unique work + KEEP policy
-     */
-    override fun onEnabled(context: Context) {
+    override fun onEnabled(context: Context?) {
         super.onEnabled(context)
-        WeatherWorker.enqueue(context)
+        CoroutineScope(Dispatchers.IO).launch {
```

Comment:
Is this actually necesary to do in the receiver? Maybe it's enough doing this on the provideContent right?

Reply:
Could you check this before merging?

Reply:
I'll double check. I think this was leftover from a refactor in progress.

Reply:
I removed, can you lgtm?